examples/model_selection/plot_roc_crossval.py

```diff
@@ -24,7 +24,7 @@
 
 .. note::
 
-    See also :func:`sklearn.metrics.auc_score`,
+    See also :func:`metrics.auc_score`,
              :func:`sklearn.model_selection.cross_val_score`,
```

Comment:
Adding `sklearn` is required for the link to render. I'll have another look at this in the nest few days.